resources/planwise/sql/facilities.sql

```diff
@@ -12,6 +12,13 @@ SELECT
 FROM facilities
 ORDER BY name;
 
+-- :name facilities-with-types :? :*
```

Comment:
Should be `facilities-from-types`, not `with-types`. 
Also, shouldn't we also `SELECT` the `type` field in all queries?